src/components/headline.tsx

```diff
@@ -76,6 +76,11 @@ const commentStyles = css`
     padding-bottom: ${remSpace[1]};
 `;
 
+const advertisementFeatureStyles = css`
+    ${styles}
+    ${textSans}
```

Comment:
Would it be preferable to use a design system font here?

Reply:
I had a problem here as I wanted to keep the headline the same size as other articles but that would require larger textSans sizes in the design system https://theguardian.design/2a1e5182b/p/930d69-typography/b/06df96

@SiAdcock

Reply:
Would be happy to try and add that to the design system if it's useful? I imagine Dotcom could have the same problem when adding glabs

Reply:
Is the issue that you need a `34px` version of `Text Sans`, and the largest currently available is `24px`?

Reply:
yep, I could go into source and find the rem value used for that but the current way would also do that.

Reply:
I've opened [this](https://github.com/guardian/source/issues/343) on Source.